lua/nvim-tree/renderer/builder.lua

```diff
@@ -118,76 +126,93 @@ function Builder:_build_folder(node, padding, git_hl)
     foldername_hl = "NvimTreeEmptyFolderName"
   end
 
-  self:_insert_highlight(foldername_hl, offset + #icon + #git_icon, #line)
+  self:_insert_highlight(foldername_hl, fname_starts_at, fname_starts_at + #foldername)
 
   if git_hl then
-    self:_insert_highlight(git_hl, offset + #icon + #git_icon, #line)
+    self:_insert_highlight(git_hl, fname_starts_at, fname_starts_at + #foldername)
   end
 end
 
--- TODO: missing git icon for symlinks
```

Comment:
buildAdd this comment back. `git_icons_tbl` is always empty for symlinks.

I have tested that git icons for symlinks are working for `_build_symlink` by manually injecting them e.g.
```lua
  git_icons_tbl = {
    {
      hl = "NvimTreeGitDirty",
      icon = "✗"
    }
  }
```

Reply:
if you add a symlink into a git directory it will not be empty :)

Reply:
maybe add a comment
```
-- TODO: missing git icon for symlinks #1217

Reply:
git icons are well present for symlinks.